content/go/concepts/strings/terms/cutprefix/cutprefix.md

```diff
@@ -0,0 +1,79 @@
+---
+Title: 'CutPrefix()'
+Description: 'Checks if a given prefix exists on the specified string. Returns the provided string without the prefix and true if found or the provided string and false if the string does not have the prefix.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`CutPrefix()`** function in Go is used to return a specified string without a given prefix string. The function returns the specified string without the given prefix and `true` if the given string has the specified prefix, otherwise it returns the full string and `false`. If the given prefix is an empty string, the function returns the full string and `true`.
+
+## Syntax
+
+```pseudo
+strings.CutPrefix(str, prefix)
+```
+
+Where `str` is the given string and `prefix` is the prefix to remove.
+
+## Example
+
+The following example demonstrates the use of the `strings.CutPrefix()` function.
+
+```go
+package main
+import (
+ "fmt"
+ "strings"
+)
+
+func main() {
+  str := "Codecademy"
+  firstPrefix := "Code"
+  secondPrefix := "Hello"
+  thirdPrefix := ""
+
+  fmt.Println(strings.CutPrefix(str, firstPrefix))
+  fmt.Println(strings.CutPrefix(str, secondPrefix))
+  fmt.Println(strings.CutPrefix(str, thirdPrefix))
+}
+```
+
+The output will be:
+
+```shell
+cademy true
+Codecademy false
+Codecademy true
+```
+
+## Codebyte Example
```

Comment:
I would suggest deleting the codebyte example because the Codecademy compiler doesn't appear to support this method (it is relatively new, introduced in go 1.2). Also the codebyte isn't demonstrating any new functionality, it is basically just duplicating the previous example

Reply:
Done! Thank you.